spec/amber/cli/commands/generate/error_template_spec.cr

```diff
@@ -0,0 +1,35 @@
+require "../../../../spec_helper"
+require "../../../../support/fixtures/cli/generators/error/*"
```

Comment:
What do you think of this file structure choice for the fixture?

Reply:
what about using the same file structure as the spec but using `_fixture` extension?

`spec/support/fixtures/cli/commands/generate/*`

Reply:
That sounds reasonable.

@eliasjpr - Can you weigh in? After spending the time that you did re-organizing the tests I definitely want your thoughts on how to structure this.

With your change, will I even need line 2, or will the spec_helper take care of the fixtures being required?

Reply:
This is not require `require "../../../../support/fixtures/cli/generators/error/*"` as long it is being required by the fixtures file